sdcm/sct_config.py

```diff
@@ -1662,12 +1662,12 @@ def _update_environment_variables(self, replace=False):
             if opt["name"] in self and (opt["env"] not in os.environ or replace):
                 os.environ[opt["env"]] = str(self[opt["name"]])
 
-    def get(self, key: str):
+    def get(self, key: str, default: Any | None = None):
```

Comment:
I propose to move this change to different PR where we can discuss it. After merge rebase your branch.